src/store/index.js

```diff
@@ -35,6 +36,9 @@ export default new Vuex.Store({
     },
     async setCollection({ state }, collectionName) {
       state.collectionName = collectionName
+    },
+    async showOrHideConnectionSection({ state }) {
+      state.shouldConnectionSectionBeVisible = !state.shouldConnectionSectionBeVisible
```

Comment:
I don't think this needs to be an app state — it would be better if we kept this state inside the component.